code/game/machinery/computer/dna_console.dm

```diff
@@ -189,6 +189,16 @@
 
 	return ..()
 
+
+/obj/machinery/computer/scan_consolenew/AltClick(mob/user)
+	if(diskette && user.canUseTopic(src, !issilicon(user)))

```

Comment:
try to use early returns, this implies that you instead invert these conditionals (e.g.  if(!diskette || !user.canUseTopic(src, !issilicon(user))) and then return there. instead of indenting while its not necesary